WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -3,10 +3,18 @@ class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
   def index
+    @everything_good = true
+
     @jobs_that_should_have_run_by_now = Delayed::Job.where(attempts: 0).where('created_at < ?', MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING.minutes.ago)
     @oldest_job_that_should_have_run_by_now = @jobs_that_should_have_run_by_now.order(:created_at).first
+    if @oldest_job_that_should_have_run_by_now
+      @everything_good = false
+    end
 
-    @everything_good = @oldest_job_that_should_have_run_by_now.nil?
+    @regulations_load_error = Regulation.regulations_load_error
+    if @regulations_load_error
+      @everything_good = false
+    end
```

Comment:
Oh, right, we need to handle the "false" case as well.

On further reflection, I find `blank?` a weird way of checking if something is falsey. I slightly prefer one of these 2 ways to skin this cat:

```
@everything_good &&= !@regulations_load_error.nil?
```

or

```
@everything_good = false if @regulations_load_error
```

I realize this is bikeshedding, but I'm curious what you think nonetheless. Was your preference for just getting this down to 1 line, or did you like the boolean nature of the `&&`?